cypress/integration/page.spec.js

```diff
@@ -1249,7 +1249,7 @@ describe('', () => {
           // if before each pass
         });
 
-        it('should make all todos active', () => {
+        it.only('should make all todos active', () => {
```

Comment:
```suggestion
        it('should make all todos active', () => {
```
don't forget to remove it